frontend/src/pages/GranteeRecord/index.js

```diff
@@ -68,15 +65,6 @@ export default function GranteeRecord({ match, location }) {
 
   return (
     <>
-      <Helmet>
```

Comment:
Whenever I load the profile page from the search page I see the title `Grantee Profile - - TTA Hub` If I switch the tab to `TTA History` and back to `Profile` I see `Grantee Profile - ${Grantee name} - TTA Hub`